modules/gsp-persistent/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "private_key_pem" {
+  description = "Sealed secrets private key"
+  value       = "${tls_private_key.sealed-secrets-key.private_key_pem}"
+}
+
+output "cert_pem" {
```

Comment:
Could we rename this to something like `sealed_secrets_cert_pem` so we don't need to rely on the description to determine its use?